docs/sources/datasources/elasticsearch/_index.md

```diff
@@ -19,17 +20,15 @@ weight: 325
 
 # Elasticsearch data source
```

Comment:
Reviewed this PR and my general impression is that there is an awful lot of assumed knowledge to get this working. There's tons of terminology that's not explained - or maybe it is explained in a concept topic? They aren't necessarily terms that we own per se, but I wonder if there is a way to be up front about the level of expertise required to be successful. I would give thought to using a Before you begin section where you can articulate exactly what is required of a user in terms of knowledge, skills, and abilities.

Before you begin:
- Ensure that you are familiar with how ElasticSearch is deployed in your environment, including:
   - Refresh intervals
   - This thing, 
   - that thing
- Ensure that xxx

I'm not sure what all of those things are, but providing a list to the user so that they can quickly review it before they dive in would be helpful.

Reply:
thanks for this feedback, I am going to try to incorporate this in the intro doc and add to the template I am creating